index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <title>CSS Grid</title>
+</head>
+<body>
+<div class="container">
+    <div id="item-a"><p>A</p></div>
+    <div id="item-b">
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. A alias aliquam, amet animi, aspernatur atque autem consequuntur earum eligendi facere impedit ipsam iure labore minus molestiae nihil, odit pariatur porro quisquam ratione repellendus repudiandae saepe unde. Amet asperiores assumenda cupiditate dignissimos dolore doloremque earum eligendi esse excepturi fuga id illum impedit iste itaque minus modi natus, nisi odit optio quas qui quia, quibusdam quidem quis repellendus sed unde veritatis? Ab accusamus animi architecto assumenda eos eum eveniet inventore, necessitatibus nulla odio porro velit. Amet culpa cupiditate doloribus et fuga inventore laborum? Iure magnam, praesentium. A aperiam at cum dolore dolorum eligendi enim error ex expedita, fugit hic molestiae nemo nesciunt numquam omnis perferendis quas quo quod vero vitae? Cumque dolor quia ullam. Adipisci consectetur cupiditate debitis deserunt ea eum, facilis fuga itaque, natus nihil quis ratione rem, sapiente sint sunt totam vitae voluptates! Aperiam consectetur dolor earum fuga, porro quo.</p>
+    </div>
+    <div id="item-c"><p>C</p></div>
+    <div id="item-d"><p>D</p></div>
+    <div id="item-e"><p>E</p></div>
+    <div id="item-f"><p>F</p></div>
```

Comment:
Yoo fijne kerst gehad? In de uitwerking van Nova zijn er geen p-elementen gebruikt hier. Maakt niet uit natuurlijk, geeft hetzelfde resultaat.